src/components/Badge/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+
+import { BaseProps } from '../types';
+import { StyledBadge, StyledBadgeContainer } from './styled/badge';
+
+type badgeVariant = 'default' | 'lightest' | 'outline-brand' | 'brand';
+
+interface Props extends BaseProps {
+    label?: ReactNode;
+    variant?: badgeVariant;
+    style?: object;
+}
+
+const Badge: React.FC<Props> = props => {
```

Comment:
Here we should add the component description as we do in react-rainbow project.
The component description is in the google drive document.